writeToStream.js

```diff
@@ -520,7 +535,7 @@ function writeString (stream, string) {
  * @api private
  */
 function writeNumber (stream, number) {
-  return stream.write(numCache[number])
+  return stream.write(getNumber(number))
 }
```

Comment:
I think we can replace that function, and have `numCache` ported here. We should really avoid that `getNumber()` function call.